components/learn/modules/ROOT/pages/deploy-and-interact.adoc

```diff
@@ -147,13 +147,13 @@ NOTE: Remember that local blockchains **do not** persist their state throughout
 [[interacting-via-the-cli]]
 == Interacting via the CLI
 
-With our `Box` contract <<deploying-a-smart-contract, deployed>>, we can start using it right away from the <<getting-started-with-the-cli, CLI>>.
+With our `Box` contract <<deploying-a-smart-contract, deployed>>, we can start using it right away from the CLI.
 
 === Sending Transactions
 
-`Box` 's first function, `store`, receives an integer value and stores it on the contract storage. Because this function _modifies_ the blockchain state, we need to _send a transaction_ to the contract to execute it.
+`Box` 's first function, `store`, receives an integer value and stores it in the contract storage. Because this function _modifies_ the blockchain state, we need to _send a transaction_ to the contract to execute it.
 
-This can be achieved with the CLI's `send-tx` command. Choose your `Box` contract when prompted by the CLI, and select the `store` function, with whatever value you want to send.
+This can be achieved with the CLI's `send-tx` command. Choose your `Box` contract when prompted by the CLI, and select the `store` function, with whatever integer value you want to send e.g. 5.
```

Comment:
```suggestion
This can be achieved with the CLI's `send-tx` command. Pick your `Box` contract when prompted by the CLI, select the `store` function, and choose a numeric value to send to it:
```

Regardless of the phrasing change, we're [avoiding e.g., i.e. and others](https://insidegovuk.blog.gov.uk/2016/07/20/changes-to-the-style-guide-no-more-eg-and-ie-etc/)

Reply:
Updated